docs/reference/licenses.nuget.org.md

```diff
@@ -0,0 +1,69 @@
+# licenses.nuget.org
+
+## Rationale
+
+With introduction of the [license expressions](nuspec.md#license) a requirement emerged to have a reliable service
+that would provide a reference text for individual license identifiers, exception identifiers or license expressions.
+An additional requirement for this service is to have a stable URL schema, that is not susceptible to the link rot,
+so that we can safely use it to provide backwards compatibility for older clients.
+
+Licenses.nuget.org fulfills that role. Gallery uses it to provide the license text reference for packages that
```

Comment:
`Gallery uses it`: I get the feeling that most people don't know that the "Gallery" is www.nuget.org. Can we say `NuGet` instead?

Reply:
What is "NuGet"? Specific products and scenarios are listed here that make use of the service, but there is no requirement that any product related to NuGet ecosystem must use it. Also, Gallery is not www.nuget.org. www.nuget.org is Gallery, but not vice versa.

I could drop the whole paragraph, but then the `Rationale` section would seem incomplete.

Reply:
"NuGet" is intentionally broad and refers to the entire NuGet client + nuget.org ecosystem. To me, the proposed statement does not imply requirements. I think that using less jargon at the cost of being less precise is a good tradeoff for introductory documentation.

Anyways, I don't feel strongly about this. Your original sentence is fine if you'd prefer to keep it.

Reply:
Just "NuGet" leaves out some great details that @agr added. Seems like he is enumerating how both client and server use LNO.

I think "Gallery" should change to "nuget.org". Not every gallery instance will do this due to forks, custom code, etc. We just need to speak for nuget.org.

Also, we should expand "nuget.exe pack" to the other pack scenarios like .NET CLI pack or pack-on-build in VS. Or, you could say "`nuget.exe pack` and other pack operations"

Reply:
Changed the wording to

> nuget pack or packing with other client tools set the licenseUrl element to point to

(changed `nuget.exe pack` to `nuget pack` to be consistent with [packing documentation](https://review.docs.microsoft.com/en-us/nuget/create-packages/creating-a-package?branch=pr-en-us-1281#running-nuget-pack-to-generate-the-nupkg-file))